### DISCUSSION TITLE
Content in "Following" is the same as "For You"

### DISCUSSION THREAD

**Discussion by MrSpadala** [community/community#13407] [Other Feature Feedback, Questions, & Ideas]
The news feed inside "Following" tab are exactly _the same_ inside the new "For You" tab, except the order of the items is different and the latter suggest me other people to follow.

I just don't understand how the two news feeds differ, or should differ.

**Comment by vanilla-extracts** [comment]
The only thing I was able to see is that the action on your repositories (like someone starred it) does not appear to show in the for you feed. Which is weird, it's a *for you* tab isn't it?

Well I think it's unhelpful and unnecessary.

**Comment by jfullstackdev** [comment]
hello guys, there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)
